src/models.py

```diff
@@ -42,6 +42,9 @@ class Comment(db.Model):
     time_stamp = db.Column(db.Integer, nullable = False)
     likes = db.Column(db.Integer, nullable = False)
     dislikes = db.Column(db.Integer, nullable = False)
+    # user_id = db.Column(db.Integer, db.ForeignKey('Person.user_id'), nullable = False)
```

Comment:
This was an attempt at setting up the foreign keys so I could have the commenter name in the comments rather that the  user_id.